### PR TITLE
Add basic Gatling test

### PR DESCRIPTION
**Pull request by LucasCairns**

### CODE REVIEW

File: gatling-load-testing/src/gatling/simulations/BrowseAndListenSimulation.scala [hunk 1]
```diff
@@ -5,98 +5,80 @@ import io.gatling.http.Predef._
 
 class BrowseAndListenSimulation extends Simulation {
 
-  val frontendUrl = "https://wayland-prisoner-content-hub-staging.apps.live-1.cloud-platform.service.justice.gov.uk"
-  val backendUrl = "https://cms-prisoner-content-hub-staging.apps.live-1.cloud-platform.service.justice.gov.uk"
+  val frontendUrl = sys.env.getOrElse("FRONTEND_URL", "https://wayland-prisoner-content-hub-staging.apps.live-1.cloud-platform.service.justice.gov.uk")
```
Comment (psoleckimoj) [inline gatling-load-testing/src/gatling/simulations/BrowseAndListenSimulation.scala:8]: Do we want these to be something that might mean the tests pass? Just thinking if the env vars weren't set we wouldn't know about it as the tests would still pass with these defaults?

File: gatling-load-testing/src/gatling/simulations/BrowseAndListenSimulation.scala [hunk 2]
```diff
@@ -5,98 +5,80 @@ import io.gatling.http.Predef._
...
-      http("Browse all topics")
-        .get("/topics")
-        .check(status.is(200))
-    )
+      http("Play the audio")
+        .get("${audioUrl}")
+    )   
+  }
+
+  val scn = scenario("Scenario Name")
```
Comment (psoleckimoj) [inline gatling-load-testing/src/gatling/simulations/BrowseAndListenSimulation.scala:58]: I suppose these could be stuck in an array or something so we can have multiple, or pull them in from an include? I dunno just thinking.

Comment (LucasCairns) [reply gatling-load-testing/src/gatling/simulations/BrowseAndListenSimulation.scala:27]: Moved to supports, we have reusable Page helpers

File: gatling-load-testing/src/gatling/simulations/BrowseAndListenSimulation.scala (hunk 1 above)
Comment (LucasCairns) [reply gatling-load-testing/src/gatling/simulations/BrowseAndListenSimulation.scala]: 100% on this, removed this configuration and expanded the setUp

**Comment by LucasCairns** [comment]
PR Moved to a repository 👉 https://github.com/ministryofjustice/prisoner-content-hub-performance-tests

**Pull request closed without merging**